<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemsExportDialog</class>
 <widget class="QDialog" name="ItemsExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export items</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Feeds</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Title:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="Title_">
           <property name="text">
            <string>Exported Feeds</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Save to:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="File_"/>
           </item>
           <item>
            <widget class="QComboBox" name="ExportFormat_">
             <item>
              <property name="text">
               <string notr="true">fb2</string>
              </property>
              <property name="icon">
               <iconset theme="application-x-zip-compressed-fb2"/>
              </property>
             </item>
             <item>
              <property name="text">
               <string notr="true">pdf</string>
              </property>
              <property name="icon">
               <iconset theme="application-pdf"/>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Browse_">
             <property name="text">
              <string>Browse...</string>
             </property>
             <property name="icon">
              <iconset theme="folder"/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>2</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Channels</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QTreeView" name="ChannelsTree_">
             <property name="rootIsDecorated">
              <bool>false</bool>
             </property>
             <property name="itemsExpandable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="SelectAll_">
               <property name="text">
                <string>Select all</string>
               </property>
               <property name="icon">
                <iconset theme="edit-select-all"/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="SelectNone_">
               <property name="text">
                <string>Select none</string>
               </property>
               <property name="icon">
                <iconset theme="edit-select-none"/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Items</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="LC::Util::CategorySelector" name="CategoriesSelector_" native="true"/>
           </item>
           <item>
            <widget class="QCheckBox" name="UnreadOnly_">
             <property name="text">
              <string>Only unread items</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>PDF options</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Page size:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="PageSizeBox_">
         <item>
          <property name="text">
           <string notr="true">A4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">A5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Letter</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Page margins:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QSpinBox" name="TopMargin_">
           <property name="suffix">
            <string> mm</string>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QSpinBox" name="LeftMargin_">
           <property name="suffix">
            <string> mm</string>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QSpinBox" name="RightMargin_">
           <property name="suffix">
            <string> mm</string>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="BottomMargin_">
           <property name="suffix">
            <string> mm</string>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Font:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QFontComboBox" name="PDFFont_">
         <property name="currentFont">
          <font>
           <stylestrategy>PreferAntialias</stylestrategy>
           <kerning>true</kerning>
          </font>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Base font size:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="PDFFontSize_">
         <property name="suffix">
          <string> pt</string>
         </property>
         <property name="minimum">
          <number>6</number>
         </property>
         <property name="maximum">
          <number>72</number>
         </property>
         <property name="value">
          <number>12</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="ButtonBox_">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LC::Util::CategorySelector</class>
   <extends>QWidget</extends>
   <header location="global">util/tags/categoryselector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>ButtonBox_</sender>
   <signal>accepted()</signal>
   <receiver>ItemsExportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>533</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonBox_</sender>
   <signal>rejected()</signal>
   <receiver>ItemsExportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>533</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
